lib/cc/engine/csslint.rb

```diff
@@ -83,5 +87,7 @@ def files_to_inspect
         end
       end
     end
+    class MissingAttributesError < StandardError
+    end
```

Comment:
Yes, and empty subclass is perfect. Please put it at the top of the file though and use the `MissingAttributesError = Class.new(StandardError)` syntax.